pkg/ccl/importccl/import_stmt.go

```diff
@@ -953,6 +953,18 @@ func importPlanHook(
 			return nil
 		}
 
+		// Double check that this transaction is implicit, so that
+		// we're allowed to commit.
+		if !p.ExtendedEvalContext().TxnImplicit {
```

Comment:
We check this above if not detached, and detached already early returned just above here, so I don't think this is needed?

Reply:
Yep, not needed. It was more of a sanity check just in-case we introduce a regression at the start of the plan hook to ensure that we don't commit an explicit transaction. I'm not super attached to it, and it may have been too defensive so removed it. I updated the comment with a brief explanation instead.